<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PublishPost</class>
 <widget class="QWidget" name="PublishPost">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Publish Post</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../icons/mammudon_icon.png</normalon>
   </iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="inReplyToText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Light</family>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>This is the content of the post you are replying to.</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="accountTitleHBox">
     <item>
      <widget class="QLabel" name="postIcon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../icons/new_post.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>@</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="spoilerLanguageHBox">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLineEdit" name="spoilerText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Spoiler Text/Content Warning (optional)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="postEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>What's on your mind? Write it here.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="mediaGridBox">
     <property name="spacing">
      <number>2</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="sensitiveCheckBox">
     <property name="text">
      <string>Mark Media as Sensitive</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="postOptionsHBox">
     <item>
      <widget class="QPushButton" name="attachBtn">
       <property name="toolTip">
        <string>Add an Attachment</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../icons/attachment.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="visibilityCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  }</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="contentTypeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {  }</string>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="publishHBox">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QPushButton" name="publishBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Publish</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scheduledBtn">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>../icons/scheduled.png</normalon>
        </iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spoilerText</tabstop>
  <tabstop>languageCombo</tabstop>
  <tabstop>postEdit</tabstop>
  <tabstop>sensitiveCheckBox</tabstop>
  <tabstop>attachBtn</tabstop>
  <tabstop>visibilityCombo</tabstop>
  <tabstop>contentTypeCombo</tabstop>
  <tabstop>publishBtn</tabstop>
  <tabstop>scheduledBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
